quilt3distribute/tests/data/zzz_generate_fake_dataset.py

```diff
@@ -68,6 +68,32 @@ def main():
     data = pd.DataFrame(rows)
     data.to_csv((BASE_PATH / "example.csv"), index=False)
 
+    ###############################################################################
```

Comment:
Is there a reason this can't be a fixture that `yield`s (in order for post-test cleanup)?

Reply:
I personally like that it generates the file and that it is commited with the repo. It is both a testing file but also an example of what types of dataset manifests are accepted and handled by this project.